service/static/js/rest_api.js

```diff
@@ -19,17 +19,13 @@ $(function () {
     }
 
     function update_address_form_data(res) {
-        if (res != null && res.addresses.length != 0) {
-            var address = res.addresses[0];
```

Comment:
This change is introducing a bug for customer specific functions (see line 326). Is it possible to undo the changes here and create a separate function for updating the address form given the response from an address subordinate api endpoint